components/library/libgda/libgda.p5m

```diff
@@ -0,0 +1,660 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2017 Dariusz Sendkowski
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/bin/$(MACH64)/gda-browser-5.0
+file path=usr/bin/$(MACH64)/gda-control-center-5.0
+link path=usr/bin/$(MACH64)/gda-list-config target=gda-list-config-5.0
+file path=usr/bin/$(MACH64)/gda-list-config-5.0
+link path=usr/bin/$(MACH64)/gda-list-server-op target=gda-list-server-op-5.0
+file path=usr/bin/$(MACH64)/gda-list-server-op-5.0
+link path=usr/bin/$(MACH64)/gda-sql target=gda-sql-5.0
+file path=usr/bin/$(MACH64)/gda-sql-5.0
+file path=usr/bin/$(MACH64)/gda-test-connection-5.0
+file path=usr/bin/$(MACH64)/gdaui-demo-5.0
+file path=usr/bin/gda-browser-5.0
+file path=usr/bin/gda-control-center-5.0
+link path=usr/bin/gda-list-config target=gda-list-config-5.0
+file path=usr/bin/gda-list-config-5.0
+link path=usr/bin/gda-list-server-op target=gda-list-server-op-5.0
+file path=usr/bin/gda-list-server-op-5.0
+link path=usr/bin/gda-sql target=gda-sql-5.0
+file path=usr/bin/gda-sql-5.0
+file path=usr/bin/gda-test-connection-5.0
+file path=usr/bin/gdaui-demo-5.0
+file path=etc/$(MACH64)/libgda-5.0/config
+file path=etc/$(MACH64)/libgda-5.0/sales_test.db
+file path=etc/libgda-5.0/config
+file path=etc/libgda-5.0/sales_test.db
```

Comment:
Are these actual config and db or just examples? If just examples, perhaps, move them out of /etc.